src/Console/Application.php

```diff
@@ -176,7 +178,7 @@ public function getLongVersion(): string
 
     protected function getDefaultCommands(): array
     {
-        return [new HelpCommand(), new ListCommand()];
+        return [new HelpCommand(), new ListCommand(), new CompleteCommand(), new DumpCompletionCommand()];
```

Comment:
does one need to run `php-cs-fixer completion -h` manually?
if list of commands/params/values change, do they need to re-run it after Fixer upgrade?

if so, we should document to end-user to do it